packages/blockchain/eth/src/lib/eth.sdk.spec.ts

```diff
@@ -21,4 +22,45 @@ describe('TatumEthSDK', () => {
       walletTestFactory.generateAddressFromPrivateKey(sdk.wallet, TEST_DATA.ETH)
     })
   })
+
+  xdescribe('auction', () => {
```

Comment:
disabled? 
move to separate test file please

Reply:
done.